extension/_locales/es-ES/messages.json

```diff
@@ -0,0 +1,56 @@
+{
+  "extensionName": {
+    "message": "Classical Search Bar",
```

Comment:
You would need to remove these commas, otherwise it is not a valid JSON

```diff
   "extensionName": {
-    "message": "Classical Search Bar",
+    "message": "Classical Search Bar"
   },
```